io.github.buonhobo.KatharaGUI.json

```diff
@@ -0,0 +1,64 @@
+{
+  "id": "io.github.buonhobo.KatharaGUI",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "KatharaGUI.sh",
+  "finish-args": [
+    "--device=all",
+    "--device=dri",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=/run/docker.sock",
+    "--filesystem=home",
+    "--filesystem=xdg-run/gvfsd"
+  ],
+  "modules": [
+    "python3-requirements.json",
+    "python3-pyyaml.json",
+    "python3-hatchling.json",
+    "python3-pygments.json",
+    {
+      "name": "vte",
+      "buildsystem": "meson",
+      "config-opts": ["--buildtype=release", "-Dgtk3=false", "-Dgtk4=true"],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.gnome.org/GNOME/vte.git",
+          "tag": "0.78.0"
+        }
+      ]
+    },
+    {
+      "name": "KatharaGUI",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 src/frontend.py /app/bin/KatharaGUI.py",
+        "install -Dm755 KatharaGUI.sh $FLATPAK_DEST/bin/KatharaGUI.sh",
+        "install -Dm755 src/connect.py /app/bin/connect.py",
+        "install -Dm755 io.github.buonhobo.KatharaGUI.desktop /app/share/applications/io.github.buonhobo.KatharaGUI.desktop",
+        "install -Dm755 data/KatharaGUI.svg /app/share/icons/hicolor/scalable/apps/io.github.buonhobo.KatharaGUI.svg",
+        "install -Dm755 io.github.buonhobo.KatharaGUI.metainfo.xml /app/share/metainfo/io.github.buonhobo.KatharaGUI.metainfo.xml"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/BuonHobo/KatharaGUI.git",
+          "commit": "9867f64a585dddee90686f61017f23e86e3e0437"
+        },
+        {
+          "type": "script",
+          "commands": ["bash -c 'python3 /app/bin/KatharaGUI.py'"],
+          "dest-filename": "KatharaGUI.sh"
+        },
+        {
+          "type": "inline",
+          "dest-filename": "io.github.buonhobo.KatharaGUI.desktop",
+          "contents": "[Desktop Entry]\nVersion=1.0\nType=Application\nName=KatharaGUI\nExec=KatharaGUI.sh\nComment=A simple Kathara GUI\nIcon=io.github.buonhobo.KatharaGUI\nTerminal=false\nCategories=Utility;Application;"
+        }
```

Comment:
Have a desktop file in your github repo and use that, it's a universal standard.